### ISSUE TITLE
Accessibility | Focus Problem with the logout button

### ISSUE THREAD

**Issue by f4-lex** [Frachtwerk/essencium-frontend#832]
**Describe the bug**
The logout button is not reachable by keyboard navigation. In the Desktop view, you just cannot reach it. In the mobile view, it **disappears entirely**.

**Expected behaviour**
You should be able to log out using the keyboard only.

- [ ] update `MIGRATION.md`
- [ ] update docs
